src/data_cleaning.py

```diff
@@ -2006,6 +2015,9 @@ def create_plant_ba_table(year):
     )
 
     # fill missing BA codes first based on the BA name, then utility name, then on the transmisison owner name
+    plant_ba["balancing_authority_code_eia"] = plant_ba[
```

Comment:
how big is the impact of this change on previously-released data? I assume it's pretty small, just would be nice to know for the release notes

Reply:
There is a single plant that was previously assigned to BPAT that will now be assigned correctly to GRID in 2021